Sources/FuzzilliCli/Profiles/QtjsProfile.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import Fuzzilli
+fileprivate let ForceQV4JITGenerator = CodeGenerator("ForceQV4JITGenerator", input: .function()) { b, f in 
+    guard let arguments = b.randCallArguments(for: f) else { return }
+    let start = b.loadInt(0)
+    let end = b.loadInt(100)
+    let step = b.loadInt(1)
+    b.forLoop(start, .lessThan, end, .Add, step) { _ in
+        b.callFunction(f, withArgs: arguments)
+    }
+}
+fileprivate let QV4JITVerifyTypeGenerator = CodeGenerator("QV4JITVerifyTypeGenerator", input: .anything) { b, v in
```

Comment:
This generator and the template below are V8 specific so I don't think they have much value for QTJS, do they?

Reply:
makes sense, copied them in there to try to generate some JIT crashes but should have taken a closer read!